script.js

```diff
@@ -0,0 +1,124 @@
+//First Prompt to Pick a Number>Character
+
+startGame();
+
+function startGame(){
+    var start_number = prompt("Good morrow, my fair midsummer adventurer. Please pick a number 1 through 4 (i.e. 1, 2, 3, or 4):");
+    var start_integer = parseInt(start_number);
+    while (start_integer == 0 || start_integer >= 5) {
+      start_number = prompt("Good morrow, my fair midsummer adventurer. Please pick a number 1 through 4 (i.e. 1, 2, 3, or 4):");
```

Comment:
I know we discussed this already during our one on one, but I would encourage you to try to refactor this particular function startGame to see how you could make it more DRY